static/app/utils/replays/types.tsx

```diff
@@ -193,6 +193,14 @@ export function isHydrationErrorFrame(
   return frame.category === 'replay.hydrate-error';
 }
 
+export function isBackgroundFrame(frame: ReplayFrame): frame is BreadcrumbFrame {
+  return frame && 'category' in frame && frame.category === 'app.background';
+}
+
+export function isForegroundFrame(frame: ReplayFrame): frame is BreadcrumbFrame {
```

Comment:
It could be useful if we had more specific types instead of `frame is BreadcrumbFrame`

Reply:
I get type issues when I try to use `BackgroundFrame` instead :(

Reply:
No. We can iterate on it later